### PR TITLE
Add CMakeLists.txt for the inference demo.

### PR DESCRIPTION
**Pull request by hedaoyuan**

### CODE REVIEW

File: benchmark/tool/C/FindPaddle.cmake [hunk 1]
```diff
@@ -0,0 +1,18 @@
+set(PADDLE_ROOT $ENV{PADDLE_ROOT} CACHE PATH "Paddle Path")
+find_path(PADDLE_INC NAMES capi.h PATHS ${PADDLE_ROOT}/include/paddle)
+find_library(PADDLE_LIB NAMES paddle_capi_shared PATHS ${PADDLE_ROOT}/lib/${ANDROID_ABI})
+if(PADDLE_INC AND PADDLE_LIB)
+  message("Found")
+else()
+  message(${PADDLE_ROOT})
+  message("Not Found")
```
Comment (Xreki) [inline benchmark/tool/C/FindPaddle.cmake]: Suggest to use `message(STATUS ...)`, and the message should be more detail, such as 
```
message(STATUS "Found PaddlePaddle (include: ${PADDLE_INC}; library: ${PADDLE_LIB})")
message(FATAL_ERROR "Cannot find PaddlePaddle on ${PADDLE_ROOT}")
```

File: benchmark/tool/C/FindPaddle.cmake [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+endif()
+include_directories(${PADDLE_ROOT}/include)
+set(THIRD_PARTY_LIBRARYS)
+list(APPEND THIRD_PARTY_LIBRARYS -lglog -lgflags -lprotobuf -lopenblas -lz)
+link_directories(${PADDLE_ROOT}/lib/${ANDROID_ABI})
+link_directories(${PADDLE_ROOT}/third_party/gflags/lib/${ANDROID_ABI})
+link_directories(${PADDLE_ROOT}/third_party/glog/lib/${ANDROID_ABI})
+link_directories(${PADDLE_ROOT}/third_party/protobuf/lib/${ANDROID_ABI})
+link_directories(${PADDLE_ROOT}/third_party/openblas/lib/${ANDROID_ABI})
```
Comment (Xreki) [inline benchmark/tool/C/FindPaddle.cmake]: Sometimes, there is no `openblas`.

File: benchmark/tool/C/FindPaddle.cmake (hunk 1 above)
Comment (hedaoyuan) [reply benchmark/tool/C/FindPaddle.cmake]: Done.

File: benchmark/tool/C/FindPaddle.cmake (hunk 2 above)
Comment (hedaoyuan) [reply benchmark/tool/C/FindPaddle.cmake]: Done.

**Review by Xreki — APPROVED** [review]
LGTM

**Pull request merged**